### PR TITLE
Adding Essentials  speed info into  whois

### LINKED ISSUE #2935: Adding Essentials /speed info into /whois
### Feature request

**Feature description**

The player's essentials /speed level is included in the /whois command.

**How the feature is useful**

This would make it much easier to be able to tell if the player is using a hacked client.
/whois already displays if the player is flying or has god on. This would be similar to that.
Thanks!

### PR DESCRIPTION
**Pull request by werogg**
Fixes #2935

### CODE REVIEW

**Comment by werogg** [comment]
Missing translations for other languages

File: Essentials/src/messages_en.properties [hunk 1]
```diff
@@ -601,6 +601,7 @@ whoisAFKSince=§6 - AFK\:§r {0} (Since {1})
 whoisBanned=§6 - Banned\:§r {0}
 whoisExp=§6 - Exp\:§r {0} (Level {1})
 whoisFly=§6 - Fly mode\:§r {0} ({1})
+whoisSpeed=§6 - Speed\:§r {0}
```
Comment (mdcfe) [inline Essentials/src/messages_en.properties]: Add messages to `messages.properties` only. Other language files should be edited on Crowdin (including `messages_en` and `messages_en_GB`).

Comment (werogg) [reply Essentials/src/messages_en.properties]: Fixed

**Review by pop4959 — APPROVED** [review]
Looks simple to me

**Pull request merged**